python/lsst/daf/butler/script/configValidate.py

```diff
@@ -19,11 +19,13 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from __future__ import annotations
+
 from .._butler import Butler
 from ..core import ValidationError
 
 
-def configValidate(repo, quiet, dataset_type, ignore):
+def configValidate(repo: str, quiet: bool, dataset_type: str, ignore: str) -> bool:
```

Comment:
I think `dataset_type` and `ignore` should be `list[str]`, doctring needs some fixing too.

Reply:
Indeed. The problem being that we declare the API this calls to accept `Iterable[str]` and `str` itself matches that type. Grr.